assert/assert.d.ts

```diff
@@ -1,10 +1,15 @@
-// Type definitions for assert and power-assert
-// Project: https://github.com/Jxck/assert
-// Project: https://github.com/twada/power-assert
+// Type definitions for commonjs-assert (known as "assert" on npm)
```

Comment:
We should remove ` (known as "assert" on npm)` this sentence.
FYI https://github.com/DefinitelyTyped/definition-header#format